x-pack/plugins/ml/public/application/components/ml_inference/components/configure_pipeline.tsx

```diff
@@ -0,0 +1,213 @@
+    modelId,
+    pipelineName,
+    pipelineNameError,
+    pipelineDescription,
+    targetField,
+    targetFieldError,
+  }) => {
+    const {
+      services: {
+        docLinks: { links },
+      },
+    } = useMlKibana();
+
+    const handleConfigChange = (value: string, type: string) => {
+      handlePipelineConfigUpdate({ [type]: value });
+    };
+
+    return (
+      <EuiFlexGroup>
+        <EuiFlexItem grow={3}>
+          <EuiTitle size="s">
+            <h4>
+              {i18n.translate(
+                'xpack.ml.trainedModels.content.indices.pipelines.addInferencePipelineModal.steps.configure.title',
+                { defaultMessage: 'Create a pipeline' }
+              )}
+            </h4>
+          </EuiTitle>
+          <EuiSpacer size="m" />
+          <EuiText color="subdued" size="s">
+            <p>
+              <FormattedMessage
+                id="xpack.ml.trainedModels.content.indices.pipelines.addInferencePipelineModal.steps.configure.description"
+                defaultMessage="Build a {pipeline} to use the trained data frame analytics model - {modelId} - for inference."
+                values={{
+                  modelId: <EuiCode>{modelId}</EuiCode>,
+                  pipeline: (
+                    <EuiLink external target="_blank" href={links.ingest.pipelines}>
+                      pipeline
+                    </EuiLink>
+                  ),
+                }}
+              />
+            </p>
+            <p>
+              {i18n.translate(
+                'xpack.ml.trainedModels.content.indices.pipelines.addInferencePipelineModal.steps.configure.descriptionUsePipelines',
+                {
+                  defaultMessage:
+                    "Use pipeline/_simulate or _reindex API to pass data into this pipeline. The model's predictions will be stored in the Target field.",
+                }
+              )}
+            </p>
+          </EuiText>
+        </EuiFlexItem>
+        <EuiFlexItem grow={7}>
+          <EuiPanel hasBorder={false} hasShadow={false}>
+            {/* NAME */}
+            <EuiForm component="form">
+              <EuiFormRow
+                fullWidth
+                label={i18n.translate(
+                  'xpack.ml.trainedModels.content.indices.pipelines.addInferencePipelineModal.steps.configure.nameLabel',
+                  {
+                    defaultMessage: 'Name',
+                  }
+                )}
+                helpText={
+                  !pipelineNameError && (
+                    <EuiText size="xs">
+                      {i18n.translate(
+                        'xpack.ml.trainedModels.content.indices.pipelines.addInferencePipelineModal.steps.configure.name.helpText',
+                        {
+                          defaultMessage:
+                            'Pipeline names are unique within a deployment and can only contain letters, numbers, underscores, and hyphens. This will create a pipeline named {pipelineName}.',
```

Comment:
```suggestion
                            'Pipeline names are unique within a deployment and can only contain letters, numbers, underscores, and hyphens.',
```

Reply:
Updated in 64ba3ac4ec279eb78f5f8a97d01005991ae76219